zerver/models.py

```diff
@@ -152,13 +154,17 @@ class Realm(ModelReprMixin, models.Model):
     date_created = models.DateTimeField(default=timezone.now) # type: datetime.datetime
     notifications_stream = models.ForeignKey('Stream', related_name='+', null=True, blank=True) # type: Optional[Stream]
     deactivated = models.BooleanField(default=False) # type: bool
-
+    authentication_methods_enabled = BitField(flags=AUTHENTICATION_FLAGS, default=2**len(AUTHENTICATION_FLAGS)-1) # type: BitHandler
```

Comment:
I think it also might make sense to give this a shorter name (e.g. realm.authentication_methods) -- that seems like it's probably detailed enough (and this name is really long!).